.github/workflows/docbuild.yml

```diff
@@ -21,11 +21,15 @@ jobs:
       run: |
         python -m pip install --upgrade pip
         pip install .[all]
-        conda install -c conda-forge xvfbwrapper
+        sudo apt-get install -y xvfb
+    - name: Run XVFB
+      run: Xvfb :99 -screen 0 1280x1024x24
+      background: true
```

Comment:
Did this ever work @36000? I am getting this error: https://github.com/yeatmanlab/pyAFQ/actions/runs/7734636723